locations/spiders/ecouter_voir_fr.py

```
@@ -0,0 +1,24 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider, clean_facebook
+
+
+class EcouterVoirFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "ecouter_voir_fr"
+    item_attributes = {"brand": "Écouter Voir", "brand_wikidata": "Q18414551"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)